<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dlgOnMapLoader</class>
 <widget class="QDialog" name="dlgOnMapLoader">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>536</width>
    <height>673</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SHP파일 GPKG 파일로 변환</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="5" column="0">
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <widget class="QLabel" name="lblSubWork">
       <property name="text">
        <string>현재 작업 진행상황</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QProgressBar" name="progressSubWork">
       <property name="value">
        <number>0</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="lblTgtFile">
       <property name="text">
        <string>변환된 공간정보가 저장될 지오패키지(GPKG) 폴더:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSplitter" name="splitter_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <widget class="QLineEdit" name="edtTgtFile">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QPushButton" name="btnTgtFile">
        <property name="minimumSize">
         <size>
          <width>80</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>80</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>선택...</string>
        </property>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
   <item row="3" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QPushButton" name="btnStart">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="minimumSize">
        <size>
         <width>200</width>
         <height>50</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Algerian</family>
         <pointsize>11</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>변환 시작</string>
       </property>
       <property name="default">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QLabel" name="lblSelLayer">
       <property name="text">
        <string>변환 대상 레이어 리스트:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTableWidget" name="tableLayer">
       <property name="editTriggers">
        <set>QAbstractItemView::NoEditTriggers</set>
       </property>
       <property name="showGrid">
        <bool>true</bool>
       </property>
       <property name="gridStyle">
        <enum>Qt::SolidLine</enum>
       </property>
       <attribute name="horizontalHeaderDefaultSectionSize">
        <number>100</number>
       </attribute>
       <attribute name="verticalHeaderDefaultSectionSize">
        <number>30</number>
       </attribute>
       <column>
        <property name="text">
         <string>SHP파일명</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>기존</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>추가</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>삭제</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>속성수정</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>도형수정</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>도형속성수정</string>
        </property>
       </column>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="lblSrcFile">
       <property name="text">
        <string>변환대상(SHP) 폴더:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSplitter" name="splitter">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <widget class="QLineEdit" name="edtSrcFile">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QPushButton" name="btnSrcFile">
        <property name="minimumSize">
         <size>
          <width>80</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>80</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>선택...</string>
        </property>
        <property name="default">
         <bool>true</bool>
        </property>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
   <item row="6" column="0">
    <layout class="QVBoxLayout" name="verticalLayout_5">
     <item>
      <widget class="QLabel" name="lblMainWork">
       <property name="text">
        <string>전체 작업 진행상황</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QProgressBar" name="progressMainWork">
       <property name="maximum">
        <number>100</number>
       </property>
       <property name="value">
        <number>0</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="invertedAppearance">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="7" column="0">
    <widget class="QPlainTextEdit" name="editLog">
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
